devito/operations/interpolators.py

```diff
@@ -4,6 +4,7 @@
 import sympy
 import numpy as np
 
+
```

Comment:
no need

Reply:
Not sure how that got in there. Good spot